### ISSUE TITLE
customizable stylesheet feature

### ISSUE THREAD

**Issue by Makopo** [Makopo/sublime-text-tooltip-lsl#4]
Implementation plan:

TooltipLSL.sublime-settings

```
{
    // any of the followings
    "css":"css-lines",
    "css-file": "file path for css file"
}
```

**Comment by buildersbrewery** [comment]
https://www.sublimetext.com/3dev

> Build 3116
> - Color Schemes: Added popupCss key, for defining default popup style sheets

**Comment by Makopo** [comment]
thx for information, i'll look that when i'm available

**Comment by buildersbrewery** [comment]
⚠️ This is a `[WIP]`:

[buildersbrewery/linden-scripting-language`/sublimetext/LSL/`](https://github.com/buildersbrewery/linden-scripting-language/blob/master/sublimetext/LSL)

If you use ST3119+ and add [`mdpopups`](https://github.com/facelessuser/sublime-markdown-popups) as a dependency you can even put correctly highlighted snippets in there by setting:

```json
// Preferences - User

    "mdpopups.sublime_user_lang_map":
    {
//      language
        "lsl":
        [
            [
//              extensions array
                "lsl"
            ],
            [
//              syntax file (without extension) paths array
                "LSL/syntaxes/LSL"
            ]
        ]
    },
    "mdpopups.use_sublime_highlighter": true,
```

compare:

![lsl-tooltips](https://cloud.githubusercontent.com/assets/13032004/23085413/fab3189e-f567-11e6-8ccd-da28fddc0231.gif)

/cc @XenHat

**Comment by XenHat** [comment]
Neat!

**Comment by XenHat** [comment]
Just installed bb/lsl to test, and I got this upon installation and all subsequent reloads; did I screw up? 
![image](https://cloud.githubusercontent.com/assets/980408/23085921/782c93a2-f538-11e6-958f-49b6ee96424b.png)

Edit: Ah, I see. The subfolder is not a valid self-contained sublime package. I *could* probably fix that if there is interest.

**Comment by buildersbrewery** [comment]
You'd have to pack the files in the subfolder `sublimetext/LSL` into a zip as it says in the `sublimetext` folder README. You probably packed all files for all editors...

**Comment by buildersbrewery** [comment]
As I said above, I'm currently working on it so it may break every now and then.

**Comment by XenHat** [comment]
Correct, that was my first mistake. But there probably is a way to add the
subfolder as a repository. I shall see what I can do.

**Comment by buildersbrewery** [comment]
Afaik the Github API lets you get subfolders only. At least I saw that one `github.io` page where you could download a subfolder of a user's repo via a three entries form.

* <https://kinolien.github.io/gitzip>
* <https://minhaskamal.github.io/DownGit>

**Comment by buildersbrewery** [comment]
Or instead of the pages above use `svn`:

`svn export https://github.com/author/repo/trunk/subdirectory`

**Comment by XenHat** [comment]
I decided to use the ZIP method for now. It would appear that sublime-text refuses to load the subfolder alone, invoking a JSON parsing error.

**Comment by buildersbrewery** [comment]
＠まこぽさん、何か手伝う事が有ったら是非ご連絡下さい。
